.travis.yml

```diff
@@ -83,6 +83,7 @@ before_install:
 install:
     - if [[ $MAIN_CMD == 'pycodestyle' ]]; then pip install pycodestyle; fi
     - if [[ $SETUP_CMD == build_sphinx* ]]; then pip install Sphinx; fi
+    - if [[ $SETUP_CMD == build_sphinx* ]]; then pip install scipy; fi  # Ugly kludge by JXP
```

Comment:
I'm pretty sure there is a more elegant way to do this, provided you actually understand how .travis.yml files work.  Is this something you would be able to figure out yourself?